src/crimson/os/seastore/async_cleaner.cc

```diff
@@ -1666,7 +1668,7 @@ bool SegmentCleaner::try_reserve_projected_usage(std::size_t projected_usage)
     return false;
   } else {
     ++stats.projected_count;
-    stats.projected_used_bytes_sum += stats.projected_used_bytes;
```

Comment:
I don't think this is a bug.

`projected_used_bytes_sum` is supposed to be always used with `projected_count`, so we can calculate the average `projected_used_bytes` during any period, and know the frequency at the same time.

Reply:
I see, will close this PR.